hieradata/common.yaml

```diff
@@ -1 +1,19 @@
-classes: []
+classes:
+    - ocf::apt
+    - ocf::auth
+    - ocf::autologout
+    - ocf::groups
+    - ocf::kerberos
+    - ocf::ldap
+    - ocf::locale
+    - ocf::motd
+    - ocf::munin::node
+    - ocf::networking
+    - ocf::packages
+    - ocf::puppet
+    - ocf::rootpw
+    - ocf::staff_users
+    - ocf::systemd
+    - ocf::utils
+
+packages: {}
```

Comment:
how do you suggest to do this: https://github.com/ocf/puppet/blob/789dec481116bebdf69620d0014dd059155ec8a3/hieradata/is_virtual/false.yaml

make a manifest for each package we might want to install?